portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -33,11 +36,19 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
  @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      long timestamp = System.currentTimeMillis();
```

Comment:
nit: you can probably just inline this variable unless you were planning on using it for something.

Reply:
I was planning on inlining it, but I wanted the time to be the same for all items in an individual post request.  I didn't want to call it in a for loop.

Reply:
Removed for loop.  timestamp is now inlined.